bears/vcs/git/GitCommitBear.py

```diff
@@ -236,6 +240,10 @@ def check_body(self, body,
                                     expressions in this list will be ignored.
         :param body_regex:          If provided, checks the presence of regex
                                     in the commit body.
+        :param valid_email_check:   If True, it checks if emails are in valid
+                                    format or not
+        :param email_dns_check:     If True, it checks if domains in the emails
+                                    are valid or not
         """
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/bears/vcs/git/GitCommitBear.py
+++ b/bears/vcs/git/GitCommitBear.py
@@ -274,8 +274,8 @@
             for line in body:
                 for email in re.findall(r'\S*@\S*', line):
                     email_check = re.match(r"(^[a-zA-Z0-9_.+-]"
-                                           +r"+@[a-zA-Z0-9-]+\."
-                                           +r"[a-zA-Z0-9-.]+$)", email)
+                                           + r"+@[a-zA-Z0-9-]+\."
+                                           + r"[a-zA-Z0-9-.]+$)", email)
                     # if eamil do not match regex there is no need to
                     # look up dns
                     if email_dns_check and email_check:
```